src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -70,10 +69,7 @@
     static {
         DESC_CHICKEN = new EditExpenseDescriptorBuilder().withDescription(VALID_DESCRIPTION_CHICKEN)
                 .withPrice(VALID_PRICE_CHICKEN)
-                .withTags(VALID_TAG_DISCOUNTED).build();
-        DESC_TRANSPORT = new EditExpenseDescriptorBuilder().withDescription(VALID_DESCRIPTION_TRANSPORT)
-                .withPrice(VALID_PRICE_TRANSPORT)
-                .withTags(VALID_TAG_CLAIMABLE, VALID_TAG_DISCOUNTED).build();
```

Comment:
Is there a reason for removing the Transport test?

Reply:
Also, if DESC_TRANSPORT is removed then we can remove the static final Strings above (in the fields of CommandTestUtil)